test/factories/test_result.rb

```diff
@@ -5,6 +5,6 @@
     host { association :host }
     profile { association :profile, :with_rules }
     end_time { Time.zone.now }
-    score { SecureRandom.rand(100) }
+    score { SecureRandom.rand(98) + 1 }
```

Comment:
Sometimes the random here generated a 0 or 100 score that was not working well with some of our tests.